components/icon/css/template.tsx

```diff
@@ -13,7 +13,7 @@ interface IIconProps {
   classNames?: string[];
 }
 
-export const iconColors = ['hemelblauw', 'wit', 'zwart'];
+export const iconColors = ['', 'hemelblauw', 'wit', 'zwart'];
```

Comment:
Waarom heb je de lege optie toegevoegd?

Reply:
Dit is een change van @Robbert, ik zal het even nalopen.

Reply:
De default is nu dat de icon `var(--utrecht-icon-color)` gebruikt `currentColor` en de kleur van de omgeving aanneemt, daarvoor is het color argument optioneel. Op deze manier kunnen componenten via `--utrecht-icon-color: currentColor` de kleur van de icon beinvloeden zonder de preciese selector te weten waarvan de `background-color` aangepast moet worden.

We kunnen de icon color voor stories op zich standaard instellen op hemelblauw, maar ik wil niet de indruk wekken dat een kleur kiezen verplicht is.